src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -185,15 +185,17 @@ void TilePyramid::update(const std::vector<Immutable<style::Layer::Impl>>& layer
 }
 
 // Moves all tiles to the cache except for those specified in the retain set.
-void TilePyramid::removeStaleTiles(const std::set<OverscaledTileID>& retain) {
+void TilePyramid::removeStaleTiles(const std::set<OverscaledTileID>& retain, const bool needsRelayout) {
```

Comment:
I'd just go ahead and inline `removeStaleTiles`... it's only called from the one place.